roles/firmware/templates/iiab-check-firmware

```diff
@@ -44,10 +44,12 @@ else
     echo -e "settings in /etc/iiab/local_vars.yml, please then run:"
     echo
     echo -e "  cd /opt/iiab/iiab"
-    echo -e "  sudo iiab-hotspot-off    # Sometimes nec, eg to restore 'wifi_up_down: True'"
-    echo -e "  sudo ./iiab-network      # Or, 'sudo ./runrole firmware' is SOMETIMES enough"
-    echo -e "  sudo iiab-hotspot-on     # Sometimes nec, eg to restore 'wifi_up_down: True'"
-    echo -e "  sudo poweroff\n"
+#    echo -e "  sudo iiab-hotspot-off    # Sometimes nec, eg to restore 'wifi_up_down: True'"
+#    echo -e "  sudo ./iiab-network      # Or, 'sudo ./runrole firmware' is SOMETIMES enough"
+#    echo -e "  sudo iiab-hotspot-on     # Sometimes nec, eg to restore 'wifi_up_down: True'"
+#    echo -e "  sudo poweroff\n"
+    echo -e "  sudo ./runrole firmware"
```

Comment:
1. `./runrole firmware` was insufficient when I tested it 6 months ago.  (In fact `./iiab-network` proved necessary several times, as documented on the original Line 48.  I don't know the reasons &mdash; but let's not bury this confirmed reality &mdash; unless the root cause is/was fully confirmed resolved in the past 6 months?)

2. If in the end `firmware/tasks/main.yml` really is changed to bypass `firmware/tasks/install.yml` at times, messaging will then need to be updated to communicate that the stronger `./runrole --reinstall firmware` is sometimes wanted.
   - Finally `./runrole --reinstall firmware` itself needs to download firmware whenever possible (fully reinstalling, as advertised) to avoid hairy and confusion-inducing documentation ambiguity.

I'll help.  But let's carve this kind of thing out into a completely separate ticket/PR please.  PR's with 45+ file changed (like PR #3312) are indeed important and sometimes necessary &mdash; but need to be kept narrowly scoped, serve their narrow purpose.  Preserving understandability + reviewability for testing volunteers.

(Let's use `curl iiab.io/fast.txt | sudo bash -s <PR1> <PR2> <PR3>` e.g. whenever 3 PR's indeed need to be batch-tested + reviewed together.  So each component piece remains clear, and tangibly tested as solid by volunteers offering to help.  Keeping understandable structure of what's really happening, very distinctly visible to community + all.)